dvc/command/get.py

```diff
@@ -3,14 +3,49 @@
 
 from .base import append_doc_link
 from .base import CmdBaseNoRepo
-from dvc.exceptions import DvcException
+from dvc.exceptions import DvcException, NotDvcRepoError
 
 
 logger = logging.getLogger(__name__)
 
 
 class CmdGet(CmdBaseNoRepo):
+    def _show_url(self):
+        from dvc.api import get_url
+
+        try:
+            url = get_url(
+                self.args.path, repo=self.args.url, rev=self.args.rev
+            )
+            logger.info(url)
+        except NotDvcRepoError:
```

Comment:
Hm, maybe it would make sense to move processing of this to `get_url` itself and raise an appropriate exception from there?